docs/setup/ubuntu2004.md

```diff
@@ -0,0 +1,43 @@
+# Install ROS 2 Galactic with Create 3 Messages on an Ubuntu 20.04 Machine
+
+## Before you start
+If you are running Ubuntu 20.04 natively on your machine, there is no extra setup required.
+These directions should work in a virtualized container within another operating system, as well.
+Note that there might be some network setup required if in a virtualized container; for example, RMWs seem to like running in a bridged network configuration rather than a NATted one.
```

Comment:
Should we give an example of what to do to setup a bridged network for a container if you have that info handy

Reply:
This is VM-software-dependent. For example, in VirtualBox, there's a setting in the "network" panel. I would have to try VMWare and Parallels and others to give a good answer.